pkg/cmd/server/admin/overwrite_bootstrappolicy.go

```diff
@@ -117,7 +116,7 @@ func OverwriteBootstrapPolicy(optsGetter restoptions.Getter, policyFile, createB
 		fmt.Fprintf(out, "Performing a dry run of policy overwrite:\n\n")
 	}
 
-	mapper := cmdclientcmd.ShortcutExpander{RESTMapper: kcmdutil.ShortcutExpander{RESTMapper: kapi.Registry.RESTMapper()}}
+	mapper := kapi.Registry.RESTMapper()
```

Comment:
Something broke. I don't remember what. I could check my irc logs later to see. I think I maybe chatted with @deads2k way back when.